Examples/MAX32650/UART/main.c

```diff
@@ -128,16 +128,14 @@ int main(void)
     if (error < E_NO_ERROR) {
         printf("-->Error initializing UART: %d\n", error);
         printf("-->Example Failed\n");
-
-        while (1) {}
+        return -1;
```

Comment:
Same for here.  Instead of using `return -1` we should use `return error` (in this case) or add a generic `E_FAIL` to `mxc_errors.h`.  We should avoid returning "naked" hard-coded files like 0 or -1.